test/support/server.js

```diff
@@ -417,4 +417,27 @@ app.get('/if-mod', function(req, res){
   }
 });
 
+var errAttempts = 0;
+app.get('/error/ok', function(req, res) {
```

Comment:
In CI we have multiple browsers running the test at the same time, and I think they're sharing the server, so they could interfere with each other's attempts.

It may be more robust to take a random ID from the client as an argument, and count retries per that ID.